velodrome/token-counter/deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         - --influx-password=$(INFLUXDB_ROOT_PWD)
         - --token=/github-tokens/k8s-merge-robot
         - --token=/github-tokens/k8s-ci-robot
+        - --token=/github-tokens/k8s-triage-robot
```

Comment:
@spiffxp can you also add a key for k8s-triage-robot in the `github-tokens` secret?